test/end-to-end/core.sh

```diff
@@ -519,23 +525,34 @@ os::cmd::expect_success "docker tag gcr.io/google_containers/pause ${DOCKER_REGI
 os::cmd::expect_success "docker push ${DOCKER_REGISTRY}/cache/prune"
 
 # record the storage before pruning
-registry_pod=$(oc get pod -l deploymentconfig=docker-registry --template='{{(index .items 0).metadata.name}}')
+registry_pod=$(oc get pod -n default -l deploymentconfig=docker-registry --template='{{(index .items 0).metadata.name}}')
```

Comment:
This file needs to be split up into multiple tests, i'm keeping dependencies close.